grace/evaluation/process.py

```diff
@@ -63,33 +63,17 @@ def update_graph_with_dummy_predictions(
         if all(obj_identity):
             # true positive node:
             if node["label"] > 0:
-                pred = {
-                    Prediction.LABEL: 1,
-                    Prediction.PROB_TN: pd,
-                    Prediction.PROB_TP: 1 - pd,
-                }
+                pred = Prediction(np.array([pd, 1 - pd]))
```

Comment:
so, then you can add probability unknown of 0.0 here:

```suggestion
                pred = Prediction(np.array([pd, 1 - pd, 0]))
```

Reply:
Ohhh ok, I see what you mean. Just assign the vector which equals the length of the annotation, even though we're not learning it. Implementing now.